maven-caching-extension/src/site/markdown/CACHE.md

```diff
@@ -17,154 +17,115 @@
   possible. Caching helps to cut down time drastically in such scenarios because it doesn't require to build cached
   dependencies.
 
 For cases there correctness must be ensured (eg prod builds), it is recommended to disable cache and do clean builds.
-This also allows you to validate cache correctness and reconcile cache outcomes on CI process.
-
-## Getting Started
-
-To on-board incremental Maven you need to complete several steps:
-
-* Get incremental Maven distribution
-* Add cache config in `.mvn`
-* Validate build results and iteratively adjust config to project specifics
-* Migrate CI to incremental Maven with remote cache (to get full benefit) - optional
-
-### Get incremental Maven distribution
-
-The recommended way is to add [Takari Maven Wrapper](https://github.com/takari/maven-wrapper) to your project. In that
-case `maven-wrapper.properties` should reference the latest incremental Maven distribution:
-
-```properties
-distributionUrl=https://your-server/maven-incremental.zip
-wrapperUrl=https://your-server/maven-wrapper-0.5.5.jar
-```
+Such scheme allows to validate cache correctness by reconciling outcomes of cached builds against the reference builds.
 
-Benefits of using Maven wrapper are following:
+## See also
 
-* simple distribution across workstations and CI envs
-* Maven stays compatible to your branch
-* further upgrades are simplified significantly  
-  If you refuse wrapper - then download, unzip and install it just as usual Maven. Further it will be assumed you use
-  Maven wrapper (`mvnw`)
-
-### Adding cache config
-
-Copy [default config `maven-cache-config.xml`](maven-cache-config.xml)
-to [`.mvn/`](https://maven.apache.org/configure.html) dir of your project.  
-To get overall understanding of cache machinery it is recommended to review the config and read comments. In typical
-scenario you need to adjust:
-
-* remote cache location
-* source code files glob
-* plugins reconciliation rules - add critical plugin parameters to reconciliation
-* add non-standard source code locations (most of locations discovered automatically from project and plugins config,
-  but still there might be edge cases)
-
-See also:
-
-* [Remote cache setup](CACHE-REMOTE.md) - instruction how to setup shared cache
-* [Cache How-To](CACHE-HOWTO.md) - cookbook for frequently encountered questions
+* [Getting started](CACHE-GETTING-STARTED.md) - getting starting with cache and usage manual
+* [Usage](CACHE-USAGE.md) - shared cache setup procedure
+* [Remote cache setup](CACHE-REMOTE.md) - shared cache setup procedure
+* [How-To](CACHE-HOWTO.md) - cookbook for typical scenarios
+* [Performance](CACHE-PERFORMANCE.md) - performance tuning
 * [Cache Parameters](CACHE-PARAMETERS.md) - description of supported parameters
-* Attached [sample `maven-cache-config.xml` config file](maven-cache-config.xml) and elements annotations in xsd schema. (Ctrl+Q in idea should
```

Comment:
Those links are broken and need to be fixed:
  * `CACHE-REMOTE.md` -> `REMOTE-CACHE.md`
  * `CACHE-HOWTO.md` -> `HOW-TO.md`
  * ...